scheduler/test/cook/test/compute_cluster.clj

```diff
@@ -51,6 +51,7 @@
     (is (= config (compute-cluster-config-ent->compute-cluster-config config-ent)))))
 
 (deftest test-db-config-ents
+  (setup)
```

Comment:
how does this relate to the rest of the PR?

Reply:
Unit tests need to run this to setup mount state, however, a lot omit it and piggyback on prior tests having had already run mount. I encountered this test breaking and fixed it.